contrib/coccinelle/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM docker.io/library/alpine:3.11
+
+LABEL maintainer="maintainer@cilium.io"
+
+RUN apk add curl autoconf automake gcc libc-dev ocaml ocaml-dev ocaml-ocamldoc ocaml-findlib make python bash && \
```

Comment:
I looked at [this Dockerfile](https://hub.docker.com/r/neszt/cppcheck-docker/dockerfile) and the maintainer seems to use a `-t name` option to name the list of packages, and delete them cleanly at the end of the sequence. Not sure if applies here given that some packages are not deleted, but might be worth checking how that option works.

Also in the same Dockerfile, the compiled binary is stripped and `/usr/src` is removed, to minimise the size of the Docker image. I have little knowledge of best practices for Docker, but shrinking the image size does sound like a good thing, so maybe we could make the same here?

Reply:
> the maintainer seems to use a -t name option to name the list of packages, and delete them cleanly at the end of the sequence.

Oh, that's neat! I'll use it.

> the compiled binary is stripped

That saves 3.8/64.9MB. Done.

> `/usr/src` is removed.

Doesn't seem to make a difference in our case.